R/apa_print_anova.R

```diff
@@ -133,11 +141,31 @@ apa_print.anova <- function(
   variance_table <- arrange_anova(x)
   object_heading <- attr(x, "heading")
 
+  # Some model objects do not contain the necessary information to calculate
+  # MSEs and effect-size measures. In such cases, print_anova() needs to be called
+  # with arguments es = NULL and mse = FALSE.
+
   if("apa_variance_table" %in% class(variance_table)) { # car::LeveneTest
     if(length(object_heading) == 1 && grepl("Levene", object_heading)) {
       if(!is.null(ellipsis$es)) stop("Effect sizes are not available for car::LeveneTest-objects.")
       return(print_anova(variance_table, es = NULL, mse = FALSE, ...))
     }
+    # lmerTest::anova.merModLmerTest ----
```

Comment:
Could you comment out these lines for the time being so that they show up in the next PR for me to review?

Reply:
Sure.

Reply:
Did that.